pkg/cli/admin/release/extract_tools.go

```diff
@@ -157,6 +160,7 @@ func (o *ExtractOptions) extractCommand(command string) error {
 		},
 		{
 			OS:      "linux",
+			Arch:    "release",
```

Comment:
`amd64` I expect these to be copies of the new locations, which we'll keep for 1-2 releases for backwards compatibility only.